staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -495,3 +465,20 @@ n+VroX6QDIMZzC7BGiUSrmVsu6xEbI+8/C7ecN2oCZZLMj96EXe6j+np4zmkQezc
 c1EwB7fNAiS0fWyE2RU6QAOZJ71bDpzQa4q4DxbOkYSybGPM/nqDRwovdjUnWeuM
 +vrJUjAZAPHJcvos0iylnc8E
 -----END PRIVATE KEY-----`)
+
+type buffer struct {
+	b  bytes.Buffer
+	rw sync.RWMutex
```

Comment:
You don't need to lock yourself. klog serializes in https://github.com/kubernetes/klog/blob/7b1c3184b2e1637d18b4a5b63a409de2becfc79b/klog.go#L748-L750

That means bytes.Buffer can be used directly.